synthesis/build_defs.bzl

```diff
@@ -118,5 +103,9 @@ synthesize_rtl = rule(
             executable = True,
             cfg = "exec",
         ),
+        "synth_tcl": attr.label(
```

Comment:
We should add a doc attribute to this parameter explaining the high level API of the environment variables.